pages/model/[uuid].tsx

```diff
@@ -148,6 +149,10 @@ const Model = () => {
     }
   }
 
+  const requestApprovalReset = async () => {
+    await postEndpoint(`/api/v1/version/${uuid}/reset-approvals`, {}).then((res) => res.json())
```

Comment:
We need to work out how we're handling errors, automatic retrying, etc.  Could you raise a ticket for this?

Reply:
(Can just be a one-liner to remind us on the elaboration board)